packages/react-email/src/app/preview/[...slug]/page.tsx

```diff
@@ -34,9 +40,15 @@ export default async function Page({ params }: { params: PreviewParams }) {
     });
   }
 
-  return <Preview renderingResult={emailRenderingResult} slug={slug} />;
+  // This suspense is so that this page doesn't warning with
+  // de-opt into client-side rendering on build
+  return (
+    <Suspense>
```

Comment:
should we add a fallback in here then? using a suspense without a fallback seems weird to me (I might be wrong tho)

Reply:
Yeah, I'm not sure if its going to have an impact with or without but I'll test it out. I'll try using the Home as fallback